tests/test_intf_mac.py

```diff
@@ -0,0 +1,248 @@
+from swsscommon import swsscommon
+
+import time
+import json
+
+class TestRouterInterfaceMac(object):
+    def setup_db(self, dvs):
+        self.pdb = swsscommon.DBConnector(0, dvs.redis_sock, 0)
+        self.adb = swsscommon.DBConnector(1, dvs.redis_sock, 0)
+        self.cdb = swsscommon.DBConnector(4, dvs.redis_sock, 0)
+
+    def add_ip_address(self, interface, ip):
+        tbl = swsscommon.Table(self.cdb, "INTERFACE")
+        fvs = swsscommon.FieldValuePairs([("NULL", "NULL")])
+        tbl.set(interface + "|" + ip, fvs)
+        time.sleep(2) # IPv6 netlink message needs longer time
+
+    def remove_ip_address(self, interface, ip):
+        tbl = swsscommon.Table(self.cdb, "INTERFACE")
+        tbl._del(interface + "|" + ip);
+        time.sleep(1)
+
+    def set_mac(self, interface, mac, ip):
+        tbl = swsscommon.Table(self.cdb, "INTERFACE")
+        fvs = swsscommon.FieldValuePairs([("mac_addr", mac)])
+        tbl.set(interface + "|" + ip, fvs)
+        time.sleep(2)
+        
+    def test_InterfaceSetMac(self, dvs, testlog):
+        self.setup_db(dvs)
+
+        # assign IP to interface
+        self.add_ip_address("Ethernet8", "10.0.0.4/31")
+
+        # check application database
```

Comment:
you could remove this portion of the test code before `set_mac` since this part is already covered by `test_interface.py`.

Reply:
sure, done.